README.md

```diff
@@ -31,14 +31,81 @@ using `make requirements` are kept up to date in latest `master` of those reposi
 
 If new dependencies were added in the `requirements.txt` of that
 repo that are not in the FPF PyPI mirror, then the maintainer needs
-to:
-
-1. Build those wheels using `make build-wheels`
-2. Push the tarball and wheel package of the new
-dependency to the FPF PyPI mirror using the steps described [here](https://github.com/freedomofpress/securedrop-debian-packaging-guide/issues/6).
-3. Make a PR updating the shasums and signature in this repository.
-4. Once this is done, `make requirements` can be used to update `build-requirements.txt`
-in the repository to be packaged.
+to do the following (we are taking `securedrop-client` project as example):
+
+### 1. Sync the wheels locally
+
+Sync all of the latest wheels `make syncwheels`
+
+### 2. Create updated build-requirements.txt for the project
+
+From the `securedrop-debian-packaging` directory,
+
+```
+PKG_DIR=/home/user/code/securedrop-client make requirements
+```
+
+This will create the proper `requirements.txt` file in the project directory along with the binary wheel
+hashes from our own Python package index server.
+
+If we are missing any wheels from our cache/build/server, it will let you know with a following message.
+
+```
+The following dependent wheel(s) are missing:
+pytest==3.10.1
+
+Please build the wheel by using the following command.
+	PKG_DIR=/home/user/code/securedrop-client make build-wheels
+Then sync the newly built wheels and sources to the s3 bucket.
+Also update the index HTML files accordingly and sync to s3.
+After these steps, please rerun the command again.
+```
+
+So, the next step is to build the wheels. To do this step, you will need the
+GPG key of @kushaldas and @conorsch @redshiftzero @emkll on the same user as
+the actual list of hashes will be signed by one of us.
+
+
+```
+PKG_DIR=/home/user/code/securedrop-client make build-wheels
```

Comment:
After building the wheels, I get the following message:
```
Successfully built alembic arrow certifi chardet idna mako markupsafe pathlib2 python-dateutil python-editor requests securedrop-sdk six sqlalchemy urllib3
Copying securedrop-sdk-0.0.11.tar.gz to cache ./localwheels
Copying securedrop_sdk-0.0.11-py3-none-any.whl to cache ./localwheels
./scripts/sync-sha256sums
Now you must sign the generated sha256sums.txt file.File '../sha256sums.txt.asc' exists. Overwrite? (y/N) y
./scripts/createdownloadurls.py > wheelsurls.txt
Done! Now please follow the instructions in
https://github.com/freedomofpress/securedrop-debian-packaging-guide/issues/6
to push these changes to the FPF PyPI index
```
Perhaps we should update this message to refer to this README instead, since it should become the canonical docs for maintainers to build wheels?
```